src/components/Loading/LoadingComponent.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import CircularProgress from '@mui/material/CircularProgress';
+
+function LoadingComponent() {
+  return (
+    <div style={{
```

Comment:
convert inline css to `LoadingComponent.module.css`

feel free to look at other components for consistency